### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

**Review by VictoriousRaptor — DISMISSED** [review]

**Comment by jjw24** [comment]
Just in case you are not aware @VictoriousRaptor, Crowdin changes should always be squashed so we don't end up with numerous commits.

I disabled auto merge before seeing that there is only a single change, but thought I let you know just in case anyway :)

File: Flow.Launcher/Languages/ko.xaml [hunk 1]
```diff
@@ -63,7 +71,7 @@
     <system:String x:Key="ShouldUsePinyin">항상 Pinyin 사용</system:String>
     <system:String x:Key="ShouldUsePinyinToolTip">Pinyin을 사용하여 검색할 수 있습니다. Pinyin (병음) 은 로마자 중국어 입력 방식입니다.</system:String>
     <system:String x:Key="AlwaysPreview">항상 미리보기</system:String>
-    <system:String x:Key="AlwaysPreviewToolTip">항상 미리보기 패널이 열린 상태로 Flow를 시작합니다. {0} 키로 미리보기를 on/off 합니다. </system:String>
+    <system:String x:Key="AlwaysPreviewToolTip">Always open preview panel when Flow activates. Press {0} to toggle preview.</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/ko.xaml:74]: @onesounds please check

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml [hunk 2]
```diff
@@ -1,112 +1,116 @@
...
-    <system:String x:Key="plugin_explorer_globalActionKeywordInvalid">글로벌 액션 키워드는 너무 많은 결과를 불러오게 될 수 있습니다. 특정한 액션 키워드를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_quickaccess_globalActionKeywordInvalid">빠른 접근은 글로벌 액션키워드로 사용할 수 없습니다. 특정한 액션 키워드를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_windowsSearchServiceNotRunning">윈도우색인 검색에 필요한 서비스가 실행되어 있지 않습니다.</system:String>
-    <system:String x:Key="plugin_explorer_windowsSearchServiceFix">이 문제를 해결하려면 Windows Search Service를 시작하세요. 이 경고를 제거하려면 여기를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_alternative">경고 메시지가 꺼졌습니다. 파일 및 폴더 검색을 위한 대안으로 Everything 플러그인을 설치하시겠습니까?{0}{0}Everything 플러그인을 설치하려면 '예'를 선택하고, 반환하려면 '아니오'를 선택하십시오</system:String>
+    <system:String x:Key="plugin_explorer_delete_folder_link">Are you sure you want to delete {0}?</system:String>
+    <system:String x:Key="plugin_explorer_deletefolderconfirm">Are you sure you want to permanently delete this folder?</system:String>
+    <system:String x:Key="plugin_explorer_deletefileconfirm">Are you sure you want to permanently delete this file?</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml:9]: @onesounds check translations please

File: Flow.Launcher/Languages/ko.xaml (hunk 1 above)
Comment (onesounds) [reply Flow.Launcher/Languages/ko.xaml]: Fixed.

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml (hunk 2 above)
Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml:9]: It's a lot to fix now. Ignore it in this patch. (I translated it for the test PR.) I will work on Crowdin later.

**Review by onesounds — APPROVED** [review]

**Pull request merged**